sdk/servicebus/service-bus/src/queueClient.ts

```diff
@@ -161,6 +167,15 @@ export class QueueClient implements Client {
    *
    * @returns Receiver|SessionReceiver A receiver to receive from a session in the Queue if
    * `sessionOptions` were provided. Else, a receiver to receive messages from the Queue.
+   * @returns SessionReceiver A receiver to receive from a session in the Queue.
+   * @throws {Error} Thrown if the underlying connection is already closed.
+   * @throws {Error} Thrown if an open receiver is already existing.
+   * @throws {InvalidOperationError} Thrown if the Queue has sessions enabled
+   * (in which case, use the overload of this method which takes
+   * `sessionOptions` argument)
+   * @throws {SessionCannotBeLockedError} Thrown if the Queue does not have sessions enabled (in which
+   * case do not pass the `sessionOptions` argument) or if Service Bus is not able to get a lock on
+   * the session (in which case try again after some time)
```

Comment:
Too many grammatical misses in comment above!
Please correct the comment if you can so there's clarity, will be reviewing the docs myself and see where would be a good place to add the docs.

Reply:
Looks like the individual docs on the overloads are getting picked up by TypeDocs, so I'm not sure what the issue is - updating docs for all of them.
If we got something wrong with docs on overloads in general, that can be fixed for across SDKs and APIs

Reply:
Logged https://github.com/Azure/azure-sdk-for-js/issues/6136 with details.
I brought it up here because we find ourselves adding more text to docs that need not exist in the first place resulting in more maintenance tax for the future.

I will leave it up to you if you want to handle the above issue in this PR or create a new one.

Reply:
Got it - partly agree, but partly the docs help document the union overload usage as well for anyone reading the source. So I'm okay to address this and other API instances as part of #6136